### PR TITLE
 list_code_example commits and updates to python.md

### LINKED ISSUE #5711: Python client library: fix broken list code example formatting
Broken: https://docs.influxdata.com/influxdb/cloud-dedicated/reference/client-libraries/v3/python/

Problem in list_code_example is breaking formatting of https://docs.influxdata.com/influxdb/cloud-dedicated/reference/client-libraries/v3/

### PR DESCRIPTION
**Pull request by MeelahMe**
Closes #5711 

_Describe your proposed changes here._

- reverted previous commits to list_code_examples: java.md, javascript.md, sharp.md
- updates to comment tag line 190

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md [hunk 1]
```diff
@@ -15,45 +15,35 @@ aliases:
...
+    # This script assumes the client object is correctly configured with your database name, token, and host URL. 
+    # After the script runs, the table variable contains the data formatted as a PyArrow table.
+
     ```
+    
 ---
 
 The InfluxDB v3 [`influxdb3-python` Python client library](https://github.com/InfluxCommunity/influxdb3-python)
-integrates {{% product-name %}} write and query operations with Python scripts and applications.
+integrates {{% product-name %}} write and query operations with Python scripts and applications.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md:46]:
```suggestion
integrates {{% product-name %}} write and query operations with Python scripts and applications.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md [hunk 2]
```diff
@@ -181,22 +171,17 @@ Given that `write_client_options` isn't specified, the client uses the default [
 
 {{% code-placeholders "DATABASE_(NAME|TOKEN)" %}}
 
-<!-- Import for tests and hide from users.
-```python
+<!-- Import for tests and hide from users. -->
+<!--
 import os
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md:176]: This allows pytest-codeblocks to run `import os` for test setup while hiding the irrelevant code from users.
When tests are run, pytest-codeblocks runs any `python` and shell codeblocks, including codeblocks within HTML comments.

Comment (jstirnaman) [reply content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md:176]: After you merge my PR into your branch and it updates here, then we can merge this PR.

Comment (MeelahMe) [reply content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md:176]: I’ve pushed the updates to my branch, and the PR should now reflects the latest changes

File: content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md (hunk 1 above)
Comment (MeelahMe) [reply content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md]: Thank you for the direction! For this request I am a tad confused. Can you please explain the a change in this that you would like me to make here?

File: content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md [hunk 3]
```diff
@@ -15,41 +15,31 @@ aliases:
...
-
-  # Instantiate an InfluxDB v3 client
-
-  with InfluxDBClient3(host=f"{{< influxdb/host >}}",
-                        database=f"DATABASE_NAME",
-                        token=f"DATABASE_TOKEN",
-                        write_client_options=wco) as client:
-
-    # Write data in batches
+  # Example: Write and que data
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md:20]:
```suggestion
  # Example: Write and query data
```
My bad

**Review by jstirnaman — APPROVED** [review]
👍 I'll fix my typo and then you can merge it.

**Comment by jstirnaman** [comment]
@MeelahMe 🚀 Thanks for fixing this. Ship it!

**Pull request merged**